test/flanders/core_test.clj

```diff
@@ -1,7 +1,32 @@
 (ns flanders.core-test
   (:require [clojure.test :refer [deftest is]]
             [flanders.core :as f])
-  (:import (flanders.types EitherType)))
+  (:import (flanders.types EitherType
+                           MapType)))
+
+
+(deftest def-entity-type-test
+  (is (thrown? clojure.lang.ExceptionInfo
+               (f/def-entity-type Foo 'bad)))
+
+  (is (thrown? clojure.lang.ExceptionInfo
+               (let [bad 43]
+                 (f/def-entity-type Foo 43))))
```

Comment:
I don't think the let is needed there. You can make two tests, one directly with an Integer the other one with the `let` and the symbol.

Reply:
I think I meant to use `bad` in the example. 😄